src/Illuminate/Queue/RedisQueue.php

```diff
@@ -120,8 +119,8 @@ public function release($queue, $payload, $delay, $attempts)
     /**
      * Pop the next job off of the queue.
      *
-     * @param  string  $queue
-     * @return \Illuminate\Contracts\Queue\Job|null
+     * @param  string|null  $queue
```

Comment:
This change is good, but I think is not the only place where this need fixing. A separate PR to fix those would be greatly appreciated. :)